packages/x-charts/src/LineChart/LineHighlightElement.tsx

```diff
@@ -104,6 +96,9 @@ function LineHighlightElement(props: LineHighlightElementProps) {
       pointerEvents="none"
       ownerState={ownerState}
       className={classes.root}
+      transform={`translate(${x}, ${y})`}
```

Comment:
Unlike the CSS version, the SVG transform does not require a comma ([source](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#translate)).

```suggestion
      transform={`translate(${x} ${y})`}
```

The CSS version should also work, but since there's [this warning](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform#transform_functions) I'm not sure if it's better to be safe:

![image](https://github.com/user-attachments/assets/e1c2ded1-b915-47ab-8fa6-98923c4152e0)